scripts/upgrade

```diff
@@ -196,12 +196,7 @@ if [ -f "$final_path/data/deleted.files" ]; then
 		# Move to the dokuwiki installation folder so the "official" commands can be used without adaptation
 		cd $final_path
 
-		# This command could not remove directory
-		#grep -Ev '^($|#)' data/deleted.files | xargs -n 1 rm -vf
-		# => "rm: cannot remove 'vendor/easybook/geshi': Is a directory"
-
-		# That one works as expected
-		grep -Ev '^($|#)' data/deleted.files | xargs -n 1 rm -fr
+		grep --extended-regexp --invert-match '^($|#)' data/deleted.files | xargs --max-args=1 rm --force --dir || true
```

Comment:
That's a rm -rf executed by sudo.
If something bad happens with grep or anything else, it could have big consequences.

As we rm a list of files, the only problem that can happens now is that we try do delete a directory before the file inside it.
Which would result with an empty directory until the next upgrade. Not that bad.